boards/pyportal/Cargo.toml

```diff
@@ -19,7 +19,8 @@ version = "0.7.5"
 optional = true
 
 [dependencies.atsamd-hal]
-version = "0.21.0"
+# version = "0.21.0"
+path = "../../hal"
```

Comment:
Same here, we'd like to keep pyportal a Tier 2 boarrd

Reply:
Not sure exactly how to manage this, then. Should I have the PR to add `embassy-time` support be a stand-alone PR, then send another PR once there's a HAL release with the examples?